braindecode/preprocessing/preprocess.py

```diff
@@ -117,11 +123,19 @@ def preprocess(concat_ds, preprocessors):
         A concat of BaseDataset or WindowsDataset datasets to be preprocessed.
     preprocessors: list(Preprocessor)
         List of Preprocessor objects to apply to the dataset.
+    save_dir : str | None
+        If a string, the preprocessed data will be saved under the specified
+        directory.
+    overwrite : bool
+        When `save_dir` is provided, controls whether to overwrite existing
+        files with the same name.
+    n_jobs : int | None
+        Number of jobs for parallel execution.
 
     Returns
     -------
-    BaseConcatDataset:
-        Preprocessed dataset.
+    None | BaseConcatDataset:
+        Preprocessed dataset, if inplace is False.
```

Comment:
There is no inplace argument. I think we coupled this with `save_dir` when we created this table of different options?

Reply:
Yes, thanks for catching this.